feature-layer-update-attributes/src/main/java/com/esri/arcgisruntime/sample/featurelayerupdateattributes/MainActivity.java

```diff
@@ -147,7 +147,7 @@ public void run() {
     });
 
     mSnackbarSuccess = Snackbar
-        .make(coordinatorLayout, "Feature successfully updated", Snackbar.LENGTH_LONG)
+        .make(coordinatorLayout != null ? coordinatorLayout : null, "Feature successfully updated", Snackbar.LENGTH_LONG) // ensure layout not null
```

Comment:
@doneill I have a better fix for this, in this pull request here

https://github.com/Esri/arcgis-runtime-samples-android/pull/139

Please review.